realm/realm-library/src/androidTest/java/io/realm/RealmTests.java

```diff
@@ -4792,4 +4793,38 @@ public void getNumberOfActiveVersions() throws InterruptedException {
         } catch (IllegalStateException ignore) {
         }
     }
+
+    @Test
+    public void getCachedInstanceDoNotTriggerStrictMode() {
+        StrictMode.setThreadPolicy(new StrictMode.ThreadPolicy.Builder()
+            .detectAll()
+            .penaltyLog()
+            .penaltyDeath()
+            .build());
+        try {
+            Realm.getInstance(realmConfig).close();
+        } finally {
```

Comment:
isn't this swallowing the exception thrown by `getInstance`?

Reply:
No, it just runs whether or not an exception is thrown.